drodrpg/DRODLib/Character.cpp

```diff
@@ -2481,6 +2487,7 @@ void CCharacter::Process(
 			break;
 			case CCharacterCommand::CC_WaitForPlayerToMove:
 			{
+				if (bRoomBeingDisplayedOnly) return;
```

Comment:
Hmm...I'm wondering how to handle this one when it an "if" condition. I don't think we need to break here. Does this seem right instead:
`if (bRoomBeingDisplayedOnly || !DidPlayerMove(command, player, nLastCommand))
 STOP_COMMAND;`

Reply:
Sure, seems fine